usagov/vote-gov-drupal
# Vote-1605 -> Cypress test update

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1605](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1605&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
After doing the updates in this PR https://github.com/usagov/vote-gov-drupal/pull/246 and this will fix that issue with the 404 page no longer having the language drop-down option 

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. Start local 
3. cd into `testing`
4. run `npm run cy:test` and verify that tests are not passing
